src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -57,6 +57,16 @@
 				/>
 			</div>
 		</section>
+		<section class="tw-my-4">
```

Comment:
We probably want a loader here while the recommended loans are being loaded. All users will see this carousel.

Reply:
Oh right, you created the placeholder empty loan data objects, never mind then!